Components/Avatar/Avatar.html

```diff
@@ -0,0 +1,181 @@
+    />
+    <title>Avatar | Storm UI</title>
+  </head>
+  <body>
+    <div class="container">
+      <nav class="navbar">
+        <a class="name" href="../../index.html"><i class="fas fa-bolt"></i> STORM</a>
+        <div class="icon-container">
+          <a href="https://github.com/b0llu/storm-documentation-site/tree/developer">
+            <i class="fab fa-github icon"></i>
+          </a>
+          <a href="https://twitter.com/TheBestDhruv">
+            <i class="fab fa-twitter icon"></i>
+          </a>
+          <a>
+            <i id="toggle-theme" class="fas fa-moon icon"></i>
+          </a>
+        </div>
+      </nav>
+      <div id="sidebar-menu" class="side-bar position-hidden">
+        <div class="nested-list stick">
+          <div>
+            <h1 class="list-topic display-flex">Get Started <span class="margin-left"><i id="close-sidebar"
+                  class="fas fa-times-circle"></i></span></h1>
+            <ul class="list-subtopic">
+              <li>
+                <a class="link-hover" href="../../Documentation/Introduction.html">Introduction</a>
+              </li>
+              <li><a class="link-hover" href="../../Documentation/Color.html">Colors</a></li>
+              <li>
+                <a class="link-hover" href="../../Documentation/Typography.html">Typography</a>
+              </li>
+              <li>
+                <a class="link-hover" href="../../Documentation/Conclusion.html">Conclusion</a>
+              </li>
+            </ul>
+          </div>
+          <div>
+            <h1 class="list-topic">Components</h1>
+            <ul class="list-subtopic">
+              <li>
+                <a class="link-active" href="./Avatar.html">Avatar</a>
+              </li>
+              <li><a class="link-hover" href="../Badge/Badge.html">Badge</a></li>
+              <li><a class="link-hover" href="../Alert/Alert.html">Alert</a></li>
+              <li><a class="link-hover" href="../Button/Button.html">Button</a></li>
+              <li><a class="link-hover" href="../Card/card.html">Card</a></li>
+              <li><a class="link-hover" href="../Image/image.html">Image</a></li>
+              <li><a class="link-hover" href="../Input/input.html">Input</a></li>
+              <li><a class="link-hover" href="../text-utilites/text-utilites.html">Text Utilities</a></li>
+              <li><a class="link-hover" href="../lists/lists.html">Lists</a></li>
+              <li><a class="link-hover" href="../navigation/navigation.html">Navigation</a></li>
+              <li><a class="link-hover" href="../modal/modal.html">Modal</a></li>
+              <li><a class="link-hover" href="../rating/rating.html">Rating</a></li>
+              <li><a class="link-hover" href="../toast/toast.html">Toast</a></li>
+              <li><a class="link-hover" href="../grid/grid.html">Grid System</a></li>
+            </ul>
+          </div>
+        </div>
+        <div></div>
+      </div>
+      <div class="content">
```

Comment:
You can give more descriptive class name here

Reply:
There is also one empty div

Reply:
sure will think on that